src/extractor/guidance/intersection_handler.cpp

```diff
@@ -37,13 +38,41 @@ IntersectionHandler::IntersectionHandler(const util::NodeBasedDynamicGraph &node
 {
 }
 
+bool IntersectionHandler::SuppressModeNavigation(const EdgeID &via_edge,
+                                                 const EdgeID &connected,
+                                                 const SuppressModeListT &SUPPRESS_MODE_LIST) const
```

Comment:
I believe it would be cleaner to not pass the mode list as a parameter.
Since you already define it globally in `travel_mode.hpp`, you are essentially shadowing the name.

Shadowing describes a situation where you use the same name (here SUPPRESS_MODE_LIST) for two variables that are available at the same time. (https://en.wikipedia.org/wiki/Variable_shadowing). Simply accessing the global variable would be cleaner, in my opinion, since I don't see a reason where we would require greater flexibility at some point.